tests/views_integration/README.md

```diff
@@ -316,10 +316,155 @@ This second test example shows what is, probably, the main feature of data-drive
 * we manipulate data as the full View, in production, would do
 
 ### Snapshot testing
-@todo @juanfra
+
+This type of testing answers the following question:
+
+> Is this View, or View partial, rendering the correct markup (HTML structure, attributes, and data output) given a specific set of template variables?
+
+On the initial test, snapshot testing will create a snapshot of the HTML markup that is output by the view or partial. Each subsequent test will be compared against the initial snapshot.
+
+Below is an example of List View:
+
+```php
+<?php
+
+namespace Tribe\Events\Views\V2\Views;
+
+use Spatie\Snapshots\MatchesSnapshots;
+use Tribe\Events\Views\V2\View;
+use Tribe\Test\Products\WPBrowser\Views\V2\ViewTestCase;
+
+class List_ViewTest extends ViewTestCase {
+
+	use MatchesSnapshots;
+
+	/**
+	 * Test render empty
+	 */
+	public function test_render_empty() {
+		// Sanity check
+		$this->assertEmpty( tribe_events()->found() );
+
+		$context = tribe_context()->alter(
+			[
+				'today'      => $this->mock_date_value,
+				'now'        => $this->mock_date_value,
+			]
+		);
+
+		$list_view = View::make( List_View::class, $context );
+		$html      = $list_view->get_html();
+
+		// Let's make sure the View is displaying what events we expect it to display.
+		$expected_post_ids = [];
+		$this->assertEquals(
+			$expected_post_ids,
+			$list_view->found_post_ids()
+		);
+
+		$this->assertMatchesSnapshot( $html );
+	}
+
+	/**
+	 * Test render with upcoming events
+	 */
+	public function test_render_with_upcoming_events() {
+		$events = [];
+
+		// Create the events.
+		foreach (
+			[
+				'tomorrow 9am',
+				'+1 week',
+				'+9 days',
+			] as $start_date
+		) {
+			$events[] = tribe_events()->set_args( [
+				'start_date' => $start_date,
+				'timezone'   => 'Europe/Paris',
+				'duration'   => 3 * HOUR_IN_SECONDS,
+				'title'      => 'Test Event - ' . $start_date,
+				'status'     => 'publish',
+			] )->create();
+		}
+		// Sanity check
+		$this->assertEquals( 3, tribe_events()->where( 'ends_after', 'now' )->count() );
+
+		$this->remap_posts( $events, [
+			'events/featured/1.json',
+			'events/single/1.json',
+			'events/single/2.json'
+		] );
+
+		$list_view = View::make( List_View::class );
+		$list_view->set_context( tribe_context()->alter( [
+			'today'      => $this->mock_date_value,
+			'now'        => $this->mock_date_value,
+			'events_per_page' => 2,
+		] ) );
+		$html = $list_view->get_html();
+
+		// Let's make sure the View is displaying what events we expect it to display.
+		$expected_post_ids = wp_list_pluck( array_slice( $events, 0, 2 ), 'ID' );
+		$this->assertEquals(
+			$expected_post_ids,
```

Comment:
@juanfra  As above, this part of the code, lines `407` to `412`, should be removed as this is **data** testing, not snapshot testing.